pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -320,9 +336,22 @@ func (co *ConvertOptions) RunConvert() (err error) {
 				&node, projectinfo.GetEdgeWorkerLabelKey(), "false"); err != nil {
 				return
 			}
-			continue
+		} else {
+			// label node as edge node
+			var updatedNode *v1.Node
+			klog.Infof("mark %s as the edge-node", node.GetName())
+			if updatedNode, err = kubeutil.LabelNode(co.clientSet,
+				&node, projectinfo.GetEdgeWorkerLabelKey(), "true"); err != nil {
+				return
+			}
+			// mark edge node as autonomous
```

Comment:
Why are edge nodes labeled with autonomy by default？
Can user can choose to turn on or off the autonomy?

Reply:
@Peeknut Thanks for the review.
Label with autonomy by default is the current behaviour of `yurtctl convert`, and I think to enable node autonomy 
by default is reasonable, but I agree that we should add an option for customization.  
`yurtctl markautonomous` provides `--autonomous-nodes` option to let users choose the autonomous nodes.
I suggest adding this option to `yurtctl convert` too.  What do you think?

Reply:
Yes, I agree.